### DISCUSSION TITLE
The contents of the slot will never change.

### DISCUSSION THREAD

**Discussion by ttdly** [vuejs/vitepress#2064] [Q&A]
I want to insert some information in the header of the document that can change with the page. And I use `layout-slots` to do this. But the contents of the slot will never change unless I refresh the page😢 How can I implement this feature🙏
You can see the changes I made [here](https://github.com/vuejs/vitepress/commit/bc5d05183648e5940c8da34b404c4f9fe680a09d).

**Comment by brc-dd** [comment]
You need to use [`computed`](https://vuejs.org/guide/essentials/computed.html):

```ts
const title = computed(() => frontmatter.value.title)
```
